pages/map.js

```diff
@@ -7,7 +7,6 @@ import Button from "@components/Button";
 import PageHead from "@components/PageHead";
 import Page from "@components/Page";
 import Badge from "@components/Badge";
-import { getTags } from "./api/discover/tags";
```

Comment:
What about having a flag on this request, so it only returns tags for users that have a location? Then only tags would be returned for users who appear on the map